README.md

```diff
@@ -139,6 +142,10 @@ int main(int argc, char* argv[])
 }
 ```
 
+You can also use [hidtest/hidtest.c](https://github.com/libusb/hidapi/blob/master/hidtest/hidtest.c)
+as a starting point for your programmes.
```

Comment:
`programs` instead of `programmes`? I know it's a bit British vs American English but it's more fitting in this context.

Reply:
`applications` ;)